### PR TITLE
Offal holiday cooking by evelyn turncoat

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
SUMMARY: Content "Offal Cookbook and recipes"

#### Purpose of change
Add more cultural recipes that make use of Offal ingredients in a rare enough fashion that a dedicated book hunter may take a while to get it.  

#### Describe the solution
Add recipes, books, comestibles and itemgroup entries. 

#### Describe alternatives you've considered
None

### CODE REVIEW

File: data/json/recipes/food/offal_dishes.json [hunk 1]
```diff
@@ -290,5 +290,85 @@
...
+    "qualities": [ { "id": "COOK", "level": 3 }, { "id": "CUT", "level": 1 } ],
+    "tools": [ [ [ "surface_heat", 50, "LIST" ] ] ],
+    "components": [
+      [ [ "lung", 4 ] ],
+      [ [ "kidney", 4 ] ],
+      [ [ "brain", 4 ] ],
+      [ [ "flour", 12 ] ],
+      [ [ "onion", 3 ], [ "irradiated_onion", 3 ] ],
+      [
+        [ "any_butter", 4 ] ],
```
Comment (Qrox) [inline data/json/recipes/food/offal_dishes.json]:
```suggestion
        [ [ "any_butter", 4 ] ],
```

Comment (Maleclypse) [reply data/json/recipes/food/offal_dishes.json]: That shows a new error now there,  is it because I'm using a requirement list?

Comment (anothersimulacrum) [reply data/json/recipes/food/offal_dishes.json]: Yes, you need `"LIST"` as the third element in the array if you're using a requirement.

File: data/json/items/book/cooking.json [hunk 2]
```diff
@@ -442,6 +442,24 @@
     "time": "35 m",
     "fun": -1
   },
+  {
+    "id": "offalcooking",
+    "type": "BOOK",
+    "name": { "str": "Offal Holiday Cooking", "str_pl": "copies of Offal Holiday Cooking" },
+    "description": "Offal Holiday Cooking by Evelyn Turncoat is a book of holiday recipes that utilize organ meats from various animals.  For the true nose to tail cook.",
```
Comment (I-am-Erk) [inline data/json/items/book/cooking.json]:
```suggestion
    "description": "Offal Holiday Cooking, by Evelyn Turncoat, is a book of holiday recipes that utilize organ meats from various animals.  For the true nose-to-tail cook.",
```

**Pull request merged**